tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -28,7 +28,12 @@
 #
 # A map: function name => name of the argument that all outputs are view of
 
-VIEW_FUNCTIONS_WITH_METADATA_CHANGE = ['view_as_complex', 'view_as_real', '_conj', '_neg_view']
+VIEW_FUNCTIONS_WITH_METADATA_CHANGE = [
```

Comment:
@Lezcano can you please update RETURNS_VIEWS_OF_INPUT to include the newly added views?

Reply:
`matrix_H, mT, mH, adjoint`

Reply:
That sounds like the right thing to do.

Reply:
By the description in the file, it seems like `numpy_T` should also be in `RETURNS_VIEWS_OF_INPUT` rather than in `RETURNS_VIEW`...